test/dict_test.py

```diff
@@ -26,3 +27,13 @@ def test_dict_app(servicer, client):
 def test_dict_deploy(servicer, client):
     d = Dict.lookup("xyz", create_if_missing=True, client=client)
     d["xyz"] = 123
+
+
+def test_dict_ephemeral(servicer, client):
+    assert servicer.n_dict_heartbeats == 0
+    with Dict.ephemeral(client=client, _heartbeat_sleep=1) as d:
+        d["foo"] = 42
+        assert d.len() == 1
+        assert d["foo"] == 42
+        time.sleep(1.5)  # Make time for 2 heartbeats
+    assert servicer.n_dict_heartbeats == 2
```

Comment:
Is there a good way to assert that the dict actually dies as expected after the context manager exit?

Reply:
We don't actually delete the dict on exit, we just rely on a heartbeat timeout after 3600 server-side for that.

It seems nice to keep the objects around in the system for some brief period in case people want to inspect them.

I thought about adding maybe a `DictDelete` method and make it do nothing for now. No strong feelings about it.